policies.py

```diff
@@ -104,10 +104,12 @@ def __init__(self, n_rules, fp_dim=8912, k=10):
         self.n_rules = n_rules
         self.X = tf.placeholder(tf.float32, shape=(None, fp_dim), name='X')
         self.y = tf.placeholder(tf.int64, shape=(None,), name='y')
+        self.keep_prob = keep_prob
 
-        inp = tf.math.log(self.X+1)
+        # inp = tf.math.log(self.X+1)
+        inp = self.X+1
```

Comment:
Please check what should be done in MO to enable Log. Open a pull request to OpenVINO repo and use this branch in the tests to convert a model.